src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -256,6 +256,9 @@ public void addContactInformationToOrganisation(
     public ContactInformation setNewContactInformationFromRequest(ContactInformation contactInformation,
                                                                   ContactInformationCreationRequest contactInfo,
                                                                   Organisation organisation) {
+        if (!StringUtils.isBlank(contactInfo.getUprn())) {
```

Comment:
yes the above will do the same and tested in various scenarios it will store null for empty or empty with spaces as well as blank ....
Am setting Uprn only if its not blank not empty and not null otherwise it will store Null as default

Reply:
ok thanks